pkg/api/platform/v1/conversion.go

```diff
@@ -434,11 +433,19 @@ func parseLimits(r *v1alpha2.ResourceLimits) (float64, float64, error) {
 	return req, limit, nil
 }
 
-func HorizontalScaleSpecConversion(spec *v1alpha2.HorizontalScale) *capsule.HorizontalScale {
+func max[T constraints.Ordered](a, b T) T {
+	if b > a {
+		return b
+	}
+	return a
+}
+
```

Comment:
I believe go finally has a `max` and `min` function builtin

Reply:
nice, thanks!